network/dag/parser_test.go

```diff
@@ -73,6 +73,17 @@ func TestParseTransaction(t *testing.T) {
 		assert.Nil(t, tx)
 		assert.EqualError(t, err, "unable to parse transaction: failed to unmarshal jws message: required field \"signatures\" not present")
 	})
+	t.Run("error - to header has invalid type", func(t *testing.T) {
```

Comment:
You could also add the `to` header to the `OK` test. Then a 100% coverage score 🚀

Reply:
Done